examples/python-webhooks/requirements.txt

```diff
@@ -0,0 +1,16 @@
+annotated-types==0.7.0
```

Comment:
i think most of these are not required here except for Flask, miro_api and missing dotenv package

Reply:
good point! I'll re work this file.

Reply:
@janza should be ready now.